### ISSUE TITLE
grunt-ts 4.0.1 ignoring templated parameters in 'reference' option

### ISSUE THREAD

**Issue by smfeest** [TypeStrong/grunt-ts#245]
This looks like a similar issue to #220 but affecting the 'reference' option instead.

Since upgrading to 4.0.1 (from 2.0.1) the following configuration:
```
ts: {
    options: {
        noImplicitAny: true,
        target: 'es5'
    },
    app: {
        options: {
            declaration: true,
        },
        src: '<%= appTsSource %>',
        reference: '<%= appSource %>app/_references.ts',
        out: '<%= appSource %>content/scripts/main.js'
    },
}
```
Results in a 'cannot create directory <%= appTsSource %>' error.

Replacing the <%= appSource %> tag in the 'reference' option with a literal path (but leaving the tags in 'src' and 'out' fixes the problem).

**Comment by nycdotnet** [comment]
Thanks for the report.  PR #235 proposes to address this but I'm doing some deeper analysis because this was supposed to be among the fixes in 4.0.1 already.  I should have time this weekend to resolve.

**Comment by nycdotnet** [comment]
Should be fixed in #235.  I will push to npm soon.  Thanks!
